core/domain/stats_jobs_one_off.py

```diff
@@ -153,7 +153,9 @@ def reduce(exp_id, stringified_values):
                 exp_services.get_multiple_explorations_by_version(
                     exp_id, version_numbers))
         except utils.ExplorationConversionError:
-            yield "This exploration %s needs to be looked into." % (exp_id)
+            yield (
+                "ERROR: Exploration %s contains non-existent features that "
```

Comment:
Can you change this to something more direct, e.g. "Exploration could not be converted to latest schema version"?

Reply:
Done.